### PR TITLE
Pkg 1768v2

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**
# prophet v1.1.3

https://anaconda.atlassian.net/browse/PKG-1768

upstream: https://github.com/facebook/prophet/tree/v1.1.3

### CODE REVIEW

File: recipe/meta.yaml
Comment (danpetry) [inline recipe/meta.yaml]: note that on conda-forge it's `false` (lower-case). There's an error message for win `UserWarning: The environment variable 'PROPHET_REPACKAGE_CMDSTAN' is being passed through with value '<hidden>'.` Not sure if this is relevant to the build failure you're experiencing.

File: recipe/meta.yaml
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Move `pip check` from commands to `run_test.py` because of the linter warning:
```
import subprocess

subprocess.run(["pip", "check"])
```

**Review by skupr-anaconda — APPROVED** [review]
LGTM but move commands to the test file

File: recipe/meta.yaml
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml]: done

File: abs.yaml
Comment (danpetry) [inline abs.yaml]: please remove

Comment (M-Waszkiewicz-Anaconda) [reply abs.yaml]: yes, waiting for approval first

File: recipe/run_test.py
Comment (danpetry) [inline recipe/run_test.py:24]: Just wondering why the need for the `run_test.py`? E.g. in the recipe you could do
`python -c "from prophet import Prophet; assert Prophet.stan_backend.get_type() = "cmdstan"` or similar
If this is a lot of work don't worry about it, it's really just a preference, better to get it in before sprint end

**Review by danpetry — APPROVED** [review]
yep good

File: recipe/run_test.py
Comment (M-Waszkiewicz-Anaconda) [reply recipe/run_test.py:24]: not sure about that, @akabanovs  maybe can comment

Comment (skupr-anaconda) [reply recipe/run_test.py:24]: It was my initial suggestion for testing it more deeply but, yeah, it can be simplified now.

File: abs.yaml
Comment (danpetry) [reply abs.yaml]: oh I see, we uncomment this to upload to snowflake and then delete it.

**Pull request merged**
